Changelog.md

```diff
@@ -1,5 +1,8 @@
 ### 0.5.0 (unreleased)
 
+Breaking Changes:
+ * General: Signed right shift uses proper arithmetic shift, i.e. rounding towards negative infinity.
```

Comment:
Can you please better highlight that this is a semantics change that might go unnoticed?